src/test/java/reposense/template/GitTestTemplate.java

```diff
@@ -46,6 +46,7 @@ public class GitTestTemplate {
     protected static final String TEST_COMMIT_HASH_PARENT = "c5a6dc774e22099cd9ddeb0faff1e75f9cf4f151";
     protected static final String MAIN_AUTHOR_NAME = "harryggg";
     protected static final String FAKE_AUTHOR_NAME = "fakeAuthor";
+    protected static final String UNCONVENTIONAL_AUTHOR_NAME = "-invalidGitUsername_TreatedAsUnknownUser";
```

Comment:
Thanks for the contribution. May I propose changing -invalidGitUsername_TreatedAsUnknownUser to another author name value such as -unconventionalAthorName since the former is no longer invalid or treated as an unknown author with the new regex.